Library/Formula/ice.rb

```diff
@@ -18,6 +31,34 @@ class Ice < Formula
   patch :DATA
 
   def install
+    resource("berkeley-db").stage do
+
+      (Pathname.pwd).install resource("berkeley-db-patch")
+      safe_system "/usr/bin/patch", "-g", "0", "-f", "-d", Pathname.pwd, "-p1", "-i", "patch-berkeley-db-5_3_28"
+
+      # BerkeleyDB dislikes parallel builds
+      ENV.deparallelize
+      # --enable-compat185 is necessary because our build shadows
+      # the system berkeley db 1.x
+      args = %W[
+        --disable-debug
+        --prefix=#{prefix}
```

Comment:
It can go in ` libexec ` and then we can prepend the ` $PATH ` and use flags to point out the right ` DBD ` to ` ice `.